astropy/convolution/convolve.py

```diff
@@ -895,7 +895,7 @@ def convolve_models(model, kernel, mode='convolve_fft', **kwargs):
         Keyword representing which function to use for convolution.
             * 'convolve_fft' : use `~astropy.convolution.convolve_fft` function.
             * 'convolve' : use `~astropy.convolution.convolve`.
-    kwargs : dict
+    **kwargs : dict
```

Comment:
I am not sure about the `**`. Is this officially required by numpydoc standards? I mean, `kwargs` itself is a `dict`, but you use `**` to indicate you want to pass the `dict` in as `key=value` pairs. 🤷

Reply:
This comment also applies to all other similar changes to both `args` and `kwargs`.

Reply:
There is no official language in numpydoc itself, though it's the convention used in the [example they give](https://numpydoc.readthedocs.io/en/latest/example.html#example) and in the test suite.

I'm on the fence, slightly leaning toward having *, and **, as there are cases in scipy where some functions actually take options as a dict. 

Here as just `options`

https://github.com/scipy/scipy/blob/04ecc30fbe767a8d9faed8c77df438a957d712ef/scipy/optimize/_linprog.py#L169-L171 

And here as `**unknown_options`

https://github.com/scipy/scipy/blob/04ecc30fbe767a8d9faed8c77df438a957d712ef/scipy/optimize/_linprog_highs.py#L80-L86

Reply:
I think I'd lean towards `**kwargs` as well, though it wouldn't seem to make much sense to then add that it is a `dict` (which it isn't really any more with the `**`), even though that is in the example too.
In any case, best to at least standardize!

Reply:
@mhvk Agreed! I treat this exactly as ``typing`` does, where ``*args`` and ``**kwargs`` are annotated by the type of the component objects.